### DISCUSSION TITLE
Submitting a Pull Request: Prettier Config file?

### DISCUSSION THREAD

**Discussion by jkenzer** [observablehq/plot#952] [Q&A] [answered]
I hope this question isn't to naïve.

I'm looking at submitting a pull request related to [this thread on the community forum](https://talk.observablehq.com/t/using-plot-externally-unable-to-add-legend/6710/7). I have the code tested and ready to submit for consideration, however when I look at a diff, I notice that my default formatting is causing all kinds of problems when compared to the original source.

Is there a prettier config I should be using to match the style of the repo? I think I'm changing 3-4 lines of code and the diff shows 144 additions and 84 deletions:

![Diff Compare](https://user-images.githubusercontent.com/643517/175433821-a877dc79-ae55-4f9c-b139-17aef00a6735.png)

Any advice would be appreciated.

**Comment by mbostock** [comment]
We don’t currently use Prettier. However, we are considering adopting it to make it easier for others to contribute.

**Comment by jkenzer** [comment]
Thanks. I think I can pull a fresh version and turn off format on save in vscode. My changes are so minor it shouldn't be an issue.

**Comment by Fil** [comment, marked as answer]
we're now using prettier in main
